lib/package.json

```diff
@@ -32,7 +32,7 @@
     "vscode-textmate": "^7.0.0"
   },
   "scripts": {
-    "build": "rollup -c",
+    "build": "rollup -c --inlineDynamicImports",
```

Comment:
This is necessary to generate a single hunk with the theme inlined. I am not sure if it is actually ideal, though.